models/articles.model.js

```diff
@@ -15,6 +11,36 @@ exports.getArticlesByIDMod = (article_id, sort_by) => {
         queryValues.push(article_id)
     }
 
+    return db.query(sqlQuery, queryValues)
+    .then(({rows}) => {
+        return rows
+    })
+}
+
+exports.getArticlesMod = (sort_by) => {
+    const validSortBy = ['created_at']
+    const queryValues = []
+
+    if (sort_by && !validSortBy.includes(sort_by)) {
+        console.log('promise rejected')
+        return Promise.reject({ status: 400, msg: 'bad request'})
+    }
+
+    let sqlQuery = 
+    `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.article_img_url, articles.votes,
+    COUNT(comments.comment_id) AS comment_count
+    FROM articles 
+    LEFT JOIN comments 
+    ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id`
+
+    //now get rid of body ^^^
+    //get comment_count
```

Comment:
Make sure to take these comments out. How could you update this query to make sure that comment count is a number?